### PR TITLE
[ipa-4-8] ipatests: bump prci templates

### PR DESCRIPTION
**Pull request by netoarmando**
Remove all freeipa-* packages from template:
https://github.com/freeipa/freeipa-pr-ci/commit/bdd98c3b9dba2ce563535d0c91dad38b532441e8

Signed-off-by: Armando Neto <abiagion@redhat.com>

### CODE REVIEW

File: ipatests/prci_definitions/nightly_ipa-4-8_latest.yaml [hunk 1]
```diff
@@ -51,7 +51,7 @@ jobs:
         git_refspec: '{git_refspec}'
         template: &ci-ipa-4-8-latest
           name: freeipa/ci-ipa-4-8-f32
-          version: 0.0.3
+          version: 0.0.4
```
Comment (fcami) [inline ipatests/prci_definitions/nightly_ipa-4-8_latest.yaml:54]: hi @netoarmando I could be wrong but some of the tests in the nightlies might fail with this change.

**Comment by fcami** [comment]
hi @netoarmando
Could you please change the symlink to point at nightly_ipa-4-8_latest.yaml ? As noted above I have doubts about a test or two with this change.

**Comment by fcami** [comment]
I think https://github.com/freeipa/freeipa/pull/4862 will be enough. Let's wait for it to conclude.

**Comment by fcami** [comment]
ACK based on CI output

**Comment by netoarmando** [comment]
ipa-4-8:

* 5820573dc7932213ee74dc883239e2cecc8fb2eb ipatests: bump prci templates

**Pull request closed without merging**